osu.Game/Screens/OnlinePlay/Playlists/PlaylistsRoomSettingsOverlay.cs

```diff
@@ -339,9 +340,8 @@ private void apply()
 
                 Duration.Value = DurationField.Current.Value;
 
-                manager?.CreateRoom(room, onSuccess, onError);
-
                 loadingLayer.Show();
+                manager?.CreateRoom(room, onSuccess, onError);
```

Comment:
Not sure whether this ordering change does anything, although it does feel more correct...

Reply:
See https://github.com/ppy/osu/pull/15648/commits/e78c5d0858479780a8bafaac5462533f5230738d, explained in the commit message.